controllers/nodehealthcheck_controller.go

```diff
@@ -653,15 +653,17 @@ func (r *NodeHealthCheckReconciler) addWatch(remediationCR *unstructured.Unstruc
 		&source.Kind{Type: remediationCR},
 		handler.EnqueueRequestsFromMapFunc(utils.NHCByRemediationCRMapperFunc(r.Log)),
 		predicate.Funcs{
-			// we are just interested in update events for now
+			// we are just interested in update and delete events for now:
+			// update: for conditions
+			// delete: when control plane node CRs are deleted for remediation the next control plane node
 			CreateFunc:  func(_ event.CreateEvent) bool { return false },
-			DeleteFunc:  func(_ event.DeleteEvent) bool { return false },
```

Comment:
By deleting this line are we not stopping to watch for CR deletion?

Reply:
no, `true` is the default